### DISCUSSION TITLE
I have an error when opening worldserver from servers

### DISCUSSION THREAD

**Discussion by jhoancito** [TrinityCore/TrinityCore#28870] [General]
I have an error when opening the worldserver of the servers as you can see in the photo that error throws me up I just compiled it, it was successful without errors
![2](https://user-images.githubusercontent.com/85852080/226775189-57c065e6-81e2-41f6-8db9-2bfca1ef05b8.PNG)

**Comment by nbyaya** [comment]
Me too.

**Comment by shamage** [comment]
See this pls : https://github.com/TrinityCore/TrinityCore/issues/28876 

Solution : extract new db2 files
